core/dbt/events/types.py

```diff
@@ -2132,17 +2068,12 @@ def message(self) -> str:
 @dataclass
 class NodeFinished(DebugLevel, NodeInfo):
     unique_id: str
-    report_node_data: ParsedModelNode
-    run_result: RunResult
+    run_result: Dict[str, Any]
```

Comment:
what's wrong with using `RunResult` instead of a dict

Reply:
There are some complications in serializing RunResult which don't seem appropriate to pull into the logging code.

Reply:
I misremembered the issue here. It was circular imports. You had added a stubs file to address that, but mashumaro doesn't work with the fake classes.